controller/array_action/array_mediator_svc.py

```diff
@@ -101,6 +101,8 @@ def build_kwargs_from_parameters(space_efficiency, pool_name, io_group, volume_n
     cli_kwargs.update(space_efficiency_kwargs)
     if io_group:
         cli_kwargs['iogrp'] = io_group
+    if volume_group:
+        cli_kwargs['volumegroup'] = volume_group
```

Comment:
no new expected errors? we want to say if the volumegroup does not exist..

Reply:
1. how is it different than https://github.com/IBM/ibm-block-csi-driver/pull/448?
2. at least in svc - pool, io_group and volume_group could all cause the same error ([CMMVC5754E](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/controller/array_action/array_mediator_svc.py#L24)), so should we add more commands inside the expect, to keep this resolution? or should the mediator interface just not require such granularity?

Reply:
I'm not familiar with the possible exception, if you think we need no new error codes so I will resolve it.

Reply:
I could change the existing "pool doesn't exist" error to a more general "invalid argument" error.
what do you think?

Reply:
if it's indeed the same `error code` so yes, please do it.

Reply:
done